requirements/lint.txt

```diff
@@ -1,7 +1,7 @@
 -r pip.txt
 maxcdn
-astroid<1.4
-pylint<1.5
+astroid
+pylint
 prospector
-pylint-django<0.7
-pyflakes<1.2.0
```

Comment:
Believe we've run into issues with these upgrading and breaking stuff, is there a strong case for unpinning?